src/Sylius/Bundle/UiBundle/templates/macro/flags.html.twig

```diff
@@ -0,0 +1,7 @@
+{% macro fromLocaleCode(locale_code, class = 'me-2') %}
```

Comment:
I think this maccro should be moved on AdminBundle.

Reply:
Done :heavy_check_mark: